src/main/java/com/google/sps/data/Event.java

```diff
@@ -41,29 +41,26 @@ public void setAddress(Location address) {
     }
 
     public void setDate(long epoch){
-        /**
-         * 'date' is recieved and stored as epoch format (ex. 1617136856)
+        /*
+         * 'date' is converted to human readable format.
          */
-        this.epoch = epoch;
+        
+        this.date = new SimpleDateFormat("EEE, d MMM yyyy HH:mm").format(new Date (epoch*1000));
     }
 
     public String getType() {
         return this.type;
     }
 
-    public String getDetails() {
-        return this.details;
+    public String getDescription() {
+        return this.description;
     }
 
     public double getPrice() {
         return this.price;
     }
 
-    public String getDate(){
-        /*
-         * 'epoch' is returned as a human readable date string.
-         */
-        
-        return new SimpleDateFormat("EEE, d MMM yyyy HH:mm").format(new Date (epoch*1000));
```

Comment:
We never use the getters. We use gson to make the `event` object a json so epoch has to be converted to date when the `event` is created.